src/Components/Votes.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react"
+import { patchVotes } from "./Api"
+
+const Votes = ({ votes, article_id }) => {
+
+    const [updateVotes, setUpdatedVotes] = useState(votes)
+
+    const handleClick = (incVotes) => {
+        setUpdatedVotes((currentVotes) => {
+            return currentVotes + incVotes
+        })
+        patchVotes(article_id, incVotes)
+        .catch(() => {
+            setUpdatedVotes((currentVotes) => {
+                return currentVotes - incVotes
+            })
+        })
+    }
+
+
+
+
+    return(
+        <>
+    <p>Votes: {updateVotes}</p> 
+    <button onClick={() => {handleClick(1)}}>+ Vote</button>
+    <button onClick={() => {handleClick(-1)}}>- Vote</button>
+    </>
+    )
```

Comment:
One nice thing to consider is making a button `disabled` once it has been clicked to stop a user from spam clicking the votes 💡